src/blockchain/handler/refund.ts

```diff
@@ -1,11 +1,13 @@
-import Contracts from '../contracts';
+import getContracts from '../contracts';
 import { SECONDARY_NETWORKS } from '../config';
 
 export default class RefundHandler {
     async processRefunds() {
-        for (const network of Object.keys(Contracts)) {
+        const contracts = getContracts();
+
+        for (const network of Object.keys(contracts)) {
```

Comment:
you can replace for `(const network of Object.keys(contracts) )` with
for `(const network in contracts)`